user_guide/src/timeseries/parsing_dates_times.md

```diff
@@ -0,0 +1,39 @@
+# Parsing dates and times
+
+## Datatypes
+
+`Polars` inherits the following datetime datatypes from Apache Arrow:
+
+- `Date`: Date representation (e.g. 2014-07-08), internally represented as days since UNIX epoch encoded by a 32-bit signed integer.
+- `Datetime`: Datetime representation (e.g. ), internally represented as nanoseconds since UNIX
```

Comment:
We miss an example after `e.g.` 

It is internally represented as a 64 bit integer since the Unix epoch (can have different units. e.g. ns, us, ms)